src/view/loader.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { StyleSheet, View } from 'react-native';
+import Background from '../component/background';
+import { CopyText } from '../component/text';
+import { FormStretcher } from '../component/form';
+import Icon from '../component/icon';
+import { color } from '../component/style';
+
+const styles = StyleSheet.create({
+  copyTxt: {
+    flexDirection: 'row',
+    fontSize: 53,
+  },
+  boltBackground: {
+    height: 46,
+    width: 25,
+    marginLeft: 26,
+    backgroundColor: '#57038D',
+    alignItems: 'center',
+    justifyContent: 'center',
+  },
+  bolt: {
+    height: 37,
+    width: 19,
+  },
+});
+
+const LoaderView = () => (
+  <Background color={color.blackDark}>
+    <FormStretcher>
```

Comment:
Not sure `FormStretcher` is the sematics we want here. It's meant to be used for forms. If we refactor/optimize it for form use, we'll end up changing this screen, which would not be ideal.

In any case, we should always wrap views content in a scrollable `MainContent` in case the content height is bigger than the viewport.